setup.py

```diff
@@ -14,21 +15,7 @@ def read(fname):
     author_email='andy@andybak.net',
     license='BSD',
     url='https://github.com/DjangoAdminHackers/django-linkcheck',
-    packages=[
-        'linkcheck',
-        'linkcheck.management',
-        'linkcheck.management.commands',
-        'linkcheck.migrations',
-        'linkcheck.tests',
-        'linkcheck.tests.sampleapp',
-    ],
```

Comment:
Using setuptool's autodetection would also include the `templatetags` package which was missed before.
However, if nobody noticed that the templatetag was missing, we could also remove it I guess... but for the future it's probably safer to include all python packages in the repo instead of listing them explicitly...